contracts/pm/mixins/MixinReserve.sol

```diff
@@ -42,6 +42,24 @@ contract MixinReserve is MContractRegistry, MReserve {
         info.thawRound = reserve.freezeRound == 0 ? 0 : reserve.freezeRound.add(freezePeriod);
     }
 
+    /**
+     * @dev Returns the amount of funds claimable by a claimant from a reserve
+     * @param _reserveHolder Address of reserve holder
+     * @param _claimant Address of claimant
+     * @return Amount of funds claimable by `_claimant`from the reserve for `_reserveHolder`
+     */
+    function claimableReserve(address _reserveHolder, address _claimant) public view returns (uint256) {
+        ReserveManager storage manager = reserveManagers[_reserveHolder];
+        Reserve storage reserve = manager.reserve;
+        if (reserveState(_reserveHolder) == ReserveState.NotFrozen) {
+            // TODO: Check if claimant is registered in current round
+            return reserve.fundsAdded.div(bondingManager().getTranscoderPoolSize());
```

Comment:
Just for completeness perhaps we should return 0 if the value returned by `bondingManager().getTranscoderPoolSize()` is 0 (and add a test). Otherwise, this line would revert. While reverting wouldn't be that harmful if we are only calling this function using a `eth_call` JSON-RPC request, I think returning 0 in this case would better match expectations (a claimant cannot claim anything if there are no recipients registered) and would be necessary if we ever call this function within a transaction as well.